test/backend_spec/seederUnitTest.js

```diff
@@ -76,12 +51,10 @@ describe('Helper Functions', function () {
 
   // eslint-disable-next-line prefer-arrow-callback
   describe('Random Rating', function () {
-    const ratings = [0, 1, 2, 3, 4, 5];
-    it('should generate a random number between 1 to 5', function () {
+    it('should generate a random number', function () {
       const randomRating = helper.randomRating();
-      const index = ratings.indexOf(randomRating);
-      const truthiness = index > -1;
-      assert.equal(true, truthiness);
+
+      assert.equal('number', typeof randomRating);
```

Comment:
Nice work on testing your random function.  You may also test that the number is in a specific range if you want to increase test precision.

Reply:
Thanks for the suggestion! I originally had a test to check if the number is in the specific range but I thought it was a little extra so I removed it.